grakn-graph/src/main/java/ai/grakn/factory/SystemKeyspace.java

```diff
@@ -77,21 +78,61 @@
     public static final TypeLabel KEYSPACE_ENTITY = TypeLabel.of("keyspace");
     public static final TypeLabel KEYSPACE_RESOURCE = TypeLabel.of("keyspace-name");
 
-    protected final Logger LOG = LoggerFactory.getLogger(SystemKeyspace.class);
+    protected static final Logger LOG = LoggerFactory.getLogger(SystemKeyspace.class);
 
     private static final ConcurrentHashMap<String, Boolean> openSpaces = new ConcurrentHashMap<>();
-    private final InternalFactory<T> factory;
+    private static final AtomicBoolean isFactorySet = new AtomicBoolean(false);
+    private static InternalFactory factory;
 
-    public SystemKeyspace(InternalFactory<T> factory){
-        this.factory = factory;
+    private SystemKeyspace(){
+    }
+
+    /**
+     * Initialises the system keyspace for a specific running instance of engine
+     * @param engineUrl the url of engine to get the config from
+     * @param properties the properties used to initialise the keyspace
+     */
+    static void initialise(String engineUrl, Properties properties){
+        if(isFactorySet.compareAndSet(false, true)){
+            factory = FactoryBuilder.getFactory(SYSTEM_GRAPH_NAME, engineUrl, properties);
+            loadSystemOntology();
+        }
+    }
+
+    /**
+     * Initialises the system keyspace for a specific running instance of engine.
+     * This initializer is used when the first graph created is the system graph.
+     * @param internalFactory the factory to use when initialising the system graph.
+     */
+    static void initialise(InternalFactory internalFactory){
+        if(factory == null) {
+            factory = internalFactory;
+            loadSystemOntology();
+        }
+    }
```

Comment:
The answer is simple my dear Felix, it is because I am an idiot and missed that one :)